docs/installation.md

```diff
@@ -12,11 +12,11 @@ You have the following options to install the operator-builder CLI:
 Use wget to download the pre-compiled binaries:
 
 ```bash
-VERSION=v0.6.0
+VERSION=v0.7.0
```

Comment:
I thought @AndriyKalashnykov had fixed these in another branch (version diffs).  Are you needing to rebase, or did I misunderstand what Andriy did?  Just checking to make sure no rebase is needed and that I am indeed correct in that I am misunderstanding.

Reply:
@scottd018 i made changes in collection-tutorial branch, no rebase needed. looks i missed that version  ))

Reply:
ah...i see the commits now.  I think I got confused with the brew stuff you fixed last week.

Reply:
@scottd018 no worries, that was different repo